### PR TITLE
Add docker and readme

### PR DESCRIPTION
**Pull request by montera82**
Add docker and readme

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Consumer micro-service
+An example program demonstrating how a consumer service could be written in Go.
+## Setup
+```bash
+git clone github.com/rafaeljesus/srv-consumer
+cd srv-consumer
+make deps
+docker-compose up
```
Comment (rafaeljesus) [inline Readme.md]: I think we could have a `make build` and than the doccker-compose file would just run the binary [see here](https://github.com/rafaeljesus/composing-go-example/blob/master/Makefile#L13) WDYT?

File: cmd/main.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ import (
 )
 
 func main() {
-	conn, ch, err := amqp.NewConnection("amqp://guest:guest@localhost:5672")
+	conn, ch, err := amqp.NewConnection(os.Getenv("AMQP_DSN"))
```
Comment (rafaeljesus) [inline cmd/main.go]: we could check if it is empty and default to `"amqp://guest:guest@localhost:5672"`

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    command: go run cmd/main.go
+    links:
+          - rabbitmq
+    restart: on-failure
+
+  rabbitmq:
+    image: rabbitmq:alpine
+    ports:
+      - "5672:5672"
+
```
Comment (rafaeljesus) [inline docker-compose.yml]: we can remove this empty space

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  consumer:
+    image: golang:alpine
+    volumes:
+      - ./:/go/src/github.com/rafaeljesus/srv-consumer
+    working_dir: /go/src/github.com/rafaeljesus/srv-consumer
+    environment:
+      AMQP_DSN: "amqp://guest:guest@rabbitmq:5672"
+    command: go run cmd/main.go
+    links:
+          - rabbitmq
```
Comment (rafaeljesus) [inline docker-compose.yml]: I think it has more than 2 tabs:
```
links:
  - rabbitmq
```

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+version: '3'
+services:
+  consumer:
+    image: golang:alpine
```
Comment (rafaeljesus) [inline docker-compose.yml]: executing the binary the imagine could be from scratch tho

File: Readme.md (hunk 1 above)
Comment (montera82) [reply Readme.md]: added a dockerfile now and called `make build` from there

File: docker-compose.yml (hunk 5 above)
Comment (montera82) [reply docker-compose.yml]: added a multistage build, last stage will use from scratch

**Review by rafaeljesus — APPROVED** [review]

**Comment by rafaeljesus** [comment]
thank you man @montera82 🏅

**Pull request merged**